[2022-02-08 01:20:33,276] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Sample_one.connect_to_astra_db manual__2022-02-08T01:20:30.189735+00:00 [queued]>
[2022-02-08 01:20:33,292] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Sample_one.connect_to_astra_db manual__2022-02-08T01:20:30.189735+00:00 [queued]>
[2022-02-08 01:20:33,293] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-08 01:20:33,294] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-08 01:20:33,295] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-08 01:20:33,312] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): connect_to_astra_db> on 2022-02-08 01:20:30.189735+00:00
[2022-02-08 01:20:33,323] {standard_task_runner.py:52} INFO - Started process 110 to run task
[2022-02-08 01:20:33,330] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Sample_one', 'connect_to_astra_db', 'manual__2022-02-08T01:20:30.189735+00:00', '--job-id', '279', '--raw', '--subdir', 'DAGS_FOLDER/astradb.py', '--cfg-path', '/tmp/tmphtkriyy7', '--error-file', '/tmp/tmpjgslccr4']
[2022-02-08 01:20:33,335] {standard_task_runner.py:77} INFO - Job 279: Subtask connect_to_astra_db
[2022-02-08 01:20:33,450] {logging_mixin.py:109} INFO - Running <TaskInstance: Sample_one.connect_to_astra_db manual__2022-02-08T01:20:30.189735+00:00 [running]> on host 6d571ddbde8b
[2022-02-08 01:20:33,556] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=vathana
AIRFLOW_CTX_DAG_ID=Sample_one
AIRFLOW_CTX_TASK_ID=connect_to_astra_db
AIRFLOW_CTX_EXECUTION_DATE=2022-02-08T01:20:30.189735+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-08T01:20:30.189735+00:00
[2022-02-08 01:20:33,559] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,560] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,561] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,562] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,563] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,564] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,565] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,566] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,567] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,568] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,569] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,570] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-08 01:20:33,903] {astradb.py:27} WARNING - Downgrading core protocol version from 66 to 65 for 3844b209-e6f1-4eda-8605-b019293c1e08-us-east1.db.astra.datastax.com:29042:fa1d4c30-bd4e-401a-ab7e-25ee24b01028. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/cassandra/cluster.html#cassandra.cluster.Cluster.protocol_version
[2022-02-08 01:20:34,040] {astradb.py:27} WARNING - Downgrading core protocol version from 65 to 5 for 3844b209-e6f1-4eda-8605-b019293c1e08-us-east1.db.astra.datastax.com:29042:fa1d4c30-bd4e-401a-ab7e-25ee24b01028. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/cassandra/cluster.html#cassandra.cluster.Cluster.protocol_version
[2022-02-08 01:20:34,196] {libevreactor.py:370} ERROR - Closing connection <LibevConnection(140222689384400) 3844b209-e6f1-4eda-8605-b019293c1e08-us-east1.db.astra.datastax.com:29042:fa1d4c30-bd4e-401a-ab7e-25ee24b01028> due to protocol error: Error from server: code=000a [Protocol error] message="Beta version of the protocol used (5/v5-beta), but USE_BETA flag is unset"
[2022-02-08 01:20:34,199] {astradb.py:27} WARNING - Downgrading core protocol version from 5 to 4 for 3844b209-e6f1-4eda-8605-b019293c1e08-us-east1.db.astra.datastax.com:29042:fa1d4c30-bd4e-401a-ab7e-25ee24b01028. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/cassandra/cluster.html#cassandra.cluster.Cluster.protocol_version
[2022-02-08 01:20:34,580] {policies.py:292} INFO - Using datacenter 'us-east1' for DCAwareRoundRobinPolicy (via host '3844b209-e6f1-4eda-8605-b019293c1e08-us-east1.db.astra.datastax.com:29042:fa1d4c30-bd4e-401a-ab7e-25ee24b01028'); if incorrect, please specify a local_dc to the constructor, or limit contact points to local cluster nodes
[2022-02-08 01:20:35,366] {python.py:175} INFO - Done. Returned value was: None
[2022-02-08 01:20:35,384] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Sample_one, task_id=connect_to_astra_db, execution_date=20220208T012030, start_date=20220208T012033, end_date=20220208T012035
[2022-02-08 01:20:35,418] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-08 01:20:35,462] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
